pgo/cmd/create.go

```diff
@@ -60,6 +60,21 @@ var BackrestS3Region string
 var PVCSize string
 var BackrestPVCSize string
 
+// BackrestRepoPath allows the pgBackRest repo path to be defined instead of using the default
+var BackrestRepoPath string
+
+// Standby determines whether or not the cluster should be created as a standby cluster
+var Standby bool
+
+// PasswordUser specifies the password for the standard user created during cluster creation
+var PasswordUser string
```

Comment:
Per comment, we would remove this and leave in password

Reply:
Removed `PasswordUser` and now back to using `Password` for the user password.